src/components/shared/forms/editUserForm.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
```

Comment:
I don't think I'd put this in shared

I think forms should be a child of `components`

Shared is more for stuff that's meant to be building blocks, this is less of a building block and more of a finished castle.